mitosheet/src/components/taskpanes/UpdateImports/UpdateImportsPostReplayTaskpane.tsx

```diff
@@ -117,8 +117,14 @@ const UpdateImportsPostReplayTaskpane = (props: UpdateImportPostReplayTaskpanePr
                             if (_invalidImportIndexes === undefined) {
                                 return;
                             }
+
                             props.setInvalidImportMessages(_invalidImportIndexes);
 
+                            void props.mitoAPI.log('get_test_import_results', {
+                                'open_due_to_replay_error': false,
```

Comment:
I think a more clear log param is `update_imports_pre_replay` and then it's true or false

Reply:
I think `open_due_to_replay_error` because you can understand what its telling you even if you don't have the context that `update_imports_pre_replay` only occurs when there is an error. 

If you have the context, then both tell you the same thing. If you don't have the context, then `open_due_to_replay_error` gives it to you!